pkg/agent/flannel/setup.go

```diff
@@ -231,6 +247,7 @@ func createFlannelConf(nodeConfig *config.Node) error {
 		return fmt.Errorf("Cannot configure unknown flannel backend '%s'", nodeConfig.FlannelBackend)
 	}
 	confJSON = strings.ReplaceAll(confJSON, "%backend%", backendConf)
+	confJSON = strings.ReplaceAll(confJSON, "%CIDR%", nodeConfig.AgentConfig.ClusterCIDR.String())
```

Comment:
Why do you need to do this again?

Reply:
Good catch, it is not needed. Probably a copy/paste mistake